package.json

```diff
@@ -105,7 +105,7 @@
     ]
   },
   "engines": {
-    "node": ">=14.0.0",
+    "node": ">=16.0.0",
```

Comment:
It's fine if we target node 16 for our dev/release workflow, but we shouldn't stop support for node 14 right away as a lot of people are still relying on it and it's also a supported runtime for SWA and Functions.

Our test suite runs starting from Node 12+, so as long as we don't have a blocking issue for supporting older Node version I'd suggest that we keep supporting at least all runtimes supported by SWA, ie Node >= 14 IIRC